### PR TITLE
Use correct payment keys

### PR DESCRIPTION
**Pull request by dasmer**

### CODE REVIEW

**Comment by dasmer** [comment]
Closes #100

File: venmo-sdk/Categories/VENTransaction+VenmoSDK.m [hunk 1]
```diff
@@ -96,14 +83,22 @@ + (instancetype)transactionWithSignedRequestDictionary:(NSDictionary *)dictionar
...
+    if (targetDictionary[@"user"]) {
+        targetHandle = targetDictionary[@"user"][@"id"];
+    } else if (targetDictionary[@"phone"]) {
+        targetHandle = targetDictionary[@"phone"];
+    } else if (targetDictionary[@"email"]) {
+        targetHandle = targetDictionary[@"email"];
+    }
     NSInteger amount = [[cleanDictionary stringForKey:@"amount"] floatValue] * 100;
-    VENTransactionTarget *target = [[VENTransactionTarget alloc] initWithHandle:toUserID amount:amount];
+    VENTransactionTarget *target = [[VENTransactionTarget alloc] initWithHandle:targetHandle amount:amount];
```
Comment (eliperkins) [inline venmo-sdk/Categories/VENTransaction+VenmoSDK.m]: Will this fail if `targetHandle` ends up being `nil`?

**Comment by eliperkins** [comment]
One :question: !

File: venmo-sdk/Categories/VENTransaction+VenmoSDK.m (hunk 1 above)
Comment (dasmer) [inline venmo-sdk/Categories/VENTransaction+VenmoSDK.m]: no. `target.handle` will just be `nil` #yolo

**Comment by ayanonagon** [comment]
:green_apple: Thanks for fixing this @dasmer!

**Pull request merged**